src/main/java/duke/textui/TextUi.java

```diff
@@ -0,0 +1,132 @@
+package duke.textui;
+
+import java.util.Scanner;
+
+/**
+ * UI that deals with the display of data for the user once the chatbot is started up.
+ */
+public class TextUi {
+
+    /**
+     * Divider line.
+     */
+    private static final String LINE = "__________________________________________";
+    /**
+     * A scanner that scans for user input.
+     */
+    private final Scanner SCANNER;
```

Comment:
In this case consider using ```scanner``` instead of ```SCANNER``` if it is just a private final instead of private static final

Reply:
Okay